### PR TITLE
Add Brokers and Networking content to 3.11 RNs

### PR DESCRIPTION
**Pull request by ghost**
@ahardin-rh PTAL.

### CODE REVIEW

File: release_notes/ocp_3_11_release_notes.adoc [hunk 1]
```diff
@@ -169,6 +169,13 @@ the tools curated by the Kubernetes upstream development community.
 Functionality is tailored toward ISVs and development teams, versus ops teams.
 The most advanced Operators that exist are using Golang.
 
+[[ocp-311-brokers]]
+=== Brokers
+
```
Comment (ahardin-rh) [inline release_notes/ocp_3_11_release_notes.adoc]: Hey @tmorriso-rh I think that we should have a few subsections under the Brokers per the What's New presentation: one heading on the Galaxy integration (with details on that feature), then one on broker support for authenticated registries, then another on the service catalog access control.

File: release_notes/ocp_3_11_release_notes.adoc [hunk 2]
```diff
@@ -169,6 +169,13 @@ the tools curated by the Kubernetes upstream development community.
...
+Support for discovering/running APB sources published to Ansible Galaxy from
+the OpenShift Automation Broker. See xref:../architecture/service_catalog/ansible_service_broker.adoc[OpenShift Automation Broker] for more information.
```
Comment (ahardin-rh) [inline release_notes/ocp_3_11_release_notes.adoc]: The link needs to include a # and section ID

File: release_notes/ocp_3_11_release_notes.adoc (hunk 1 above)
Comment (tmorriso-rh) [reply release_notes/ocp_3_11_release_notes.adoc]: Thanks, I added those subsections.

**Comment by tmorriso-rh** [comment]
@ahardin-rh I incorporated your suggested changes. PTAL. Thanks!

File: release_notes/ocp_3_11_release_notes.adoc [hunk 3]
```diff
@@ -169,6 +169,33 @@ the tools curated by the Kubernetes upstream development community.
 Functionality is tailored toward ISVs and development teams, versus ops teams.
 The most advanced Operators that exist are using Golang.
 
+[[ocp-311-brokers]]
+=== Brokers
+
+[[ocp-311-automation-broker-ansible]]
+==== {product-title} Automation Broker Integration with Ansible Galaxy
+
+Support for discovering/running APB sources published to Ansible Galaxy from
```
Comment (ahardin-rh) [inline release_notes/ocp_3_11_release_notes.adoc]: discovering and running

File: release_notes/ocp_3_11_release_notes.adoc [hunk 4]
```diff
@@ -169,6 +169,33 @@ the tools curated by the Kubernetes upstream development community.
...
+==== Broker Support for Authenticated Registries
+
+{product-title} added support for authenticated registries. Broker uses
+`cluster-wide` settings for registry authentication credentials as the default
+for broker. You can define `oreg_auth_user` and `oreg_auth_password` in the inventory file to configure the credentials.
+
+[[ocp-311-service-catalog-access-control]]
+==== Service Catalog Access Control
+
+Broker now confirms to the Open Service Broker specification, which means you
```
Comment (ahardin-rh) [inline release_notes/ocp_3_11_release_notes.adoc]: The broker

File: release_notes/ocp_3_11_release_notes.adoc [hunk 5]
```diff
@@ -169,6 +169,33 @@ the tools curated by the Kubernetes upstream development community.
...
+can register broker with the service catalog as either a cluster-scoped
```
Comment (ahardin-rh) [inline release_notes/ocp_3_11_release_notes.adoc]: the broker

**Comment by ahardin-rh** [comment]
Just a few comments from me. Thanks!

**Comment by tmorriso-rh** [comment]
@ahardin-rh PTAL at the updates to Brokers and Networking. Thanks!

File: release_notes/ocp_3_11_release_notes.adoc [hunk 6]
```diff
@@ -169,6 +169,34 @@ the tools curated by the Kubernetes upstream development community.
...
+
+{product-title} added support for authenticated registries. The broker uses
+`cluster-wide` as the default setting for registry authentication credentials.
+You can define `oreg_auth_user` and `oreg_auth_password` in the inventory file
+to configure the credentials.
+
+[[ocp-311-service-catalog-access-control]]
+==== Service Catalog Access Control
+
+The broker now confirms to the Open Service Broker specification, which means
```
Comment (vikram-redhat) [inline release_notes/ocp_3_11_release_notes.adoc]: typo - this should be conforms.

**Pull request merged**